pages/05.android/06.solving-problems/09.background-work/docs.en.md

```diff
@@ -6,8 +6,8 @@ taxonomy:
         - docs
 ---
 
-# If AdGuard is disappearing after a while from the background work
-In some cases, apps won't stay in the background, "alive" or in a sleep mode due to Android OS optimization function or because of "battery save" mode. That can be uncomfortable for you to relaunch it each time it closes. To avoid that background termination you need to follow this steps marked for each version of Android OS. Note that this instruction is similar for mostly all different versions of Android OS.
+# How to protect AdGuard from being disabled by the system?
+In some cases, apps won't stay in the background, "alive" or in a sleep mode due to Android OS optimization function or "battery save" mode - that functions closing up background applications. That can be uncomfortable for you to relaunch it each time it closes. To avoid that background termination you need to follow this steps marked for each version of Android OS. Note that this instruction is similar for mostly all different versions of Android OS.
```

Comment:
In some cases, apps won't stay in the background ("alive" or in a sleep mode) due to Android OS optimization function, or so called "battery save mode" — this function can kill background applications. It may be inconvenient to relaunch them each time they are getting closed. To avoid the background app termination you need to follow these steps which we described separately for each manufacturer (version) of Android OS. Note that instructions for different manufacturers are mostly very similar.